Calls ldconfig which is not available and remove -march=native because it's not
portable across platforms.

Upstream: rejected, author not cooperative.

From: Fons Adriaensen <fons@linuxaudio.org>
To: David Demelier <markand@malikania.fr>
Subject: Re: [PATCH] zita-convolver: fix build on Alpine Linux
> I've packaged your zita-convolver software under Alpine Linux (which
> uses musl [1] libc). Distributions that use musl do not have ldconfig
> and even with, it's a system command that should not be called directly
> from the software IMHO. It's a packager/user issue.

Don't know what you mean by 'should not be called directly from the
software', but calling ldconfig is required on most Linux systems
when installing a shared library.

> Also, it looks like the -march=native option isn't portable, it is not
> recognized under certains architectures (s390x IIRC).

If I would remove everything that isn't supported everywhere from
zita-convolver, then it would simply not exist. 

The source code and Makefile are meant to work on most Linux 
systems. When that is not the case it is up to the packagers
to make the required changes. 

So, thanks for the work done, but these changes won't make it
into the code I make available.
--- a/Makefile	2020-04-17 23:15:45.045478833 +0200
+++ b/Makefile	2020-04-17 23:15:51.438846231 +0200
@@ -34,7 +34,6 @@
 CPPFLAGS += -I. -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS
 CPPFLAGS += -DENABLE_VECTOR_MODE 
 CXXFLAGS += -fPIC -Wall -ffast-math -funroll-loops -O2
-CXXFLAGS += -march=native
 LDLFAGS += 
 LDLIBS +=
 
@@ -56,7 +55,6 @@
 	install -d $(DESTDIR)$(LIBDIR)
 	install -m 644 $(ZITA-CONVOLVER_H) $(DESTDIR)$(INCDIR)
 	install -m 755 $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(LIBDIR)
-	ldconfig
 	ln -sf $(ZITA-CONVOLVER_MIN) $(DESTDIR)$(LIBDIR)/$(ZITA-CONVOLVER_SO)
 
 uninstall:
